layers/meta-resin-raspberrypi/recipes-bsp/bootfiles/bcm2835-bootfiles.bbappend

```diff
@@ -1,3 +1,8 @@
 # add support for the rpi2 1.2
-RPIFW_SRCREV = "6d45dcf869688163fa1c745cb059b14baf61b8fd"
-RPIFW_DATE = "20161209"
```

Comment:
I don't understand why we need this bbappend anymore? The 1.2 support of rpi2 should be currently available in the version provided by the bsp layer.